docs/integrations/sources/salesforce.md

```diff
@@ -734,6 +734,8 @@ List of available streams:
 
 | Version | Date | Pull Request | Subject |
 | :--- | :--- | :--- | :--- |
+
+| 0.1.6 | 2021-11-16 | [8009](https://github.com/airbytehq/airbyte/pull/8009) | Fix retring of BULK jobs |
```

Comment:
why does this say 0.1.6 but the standard source definition say 0.1.5?

Reply:
sure. I'll do it